spec/web/controllers/users/registration_spec.rb

```diff
@@ -14,6 +14,16 @@ def call
   UserStruct = Struct.new('User', :user)
 end
 
+# class UserRepoFake
+#   def initialize(users = [])
+#     @users = users
+#   end
+
+#   def by_email(email)
+#     @users.select { |user| user.fetch(:email) == email }
+#   end
+# end
+
```

Comment:
Commented out because the `UserRepo` is used directly in the controllers for checking email uniqueness  (see comment above)